packages/amplication-client/src/Plugins/PluginsCatalog.tsx

```diff
@@ -13,9 +19,15 @@ type Props = AppRouteProps & {
     resource: string;
   }>;
 };
-
+const DIALOG_CLASS_NAME = "limitation-dialog";
 const PluginsCatalog: React.FC<Props> = ({ match }: Props) => {
   const { resource } = match.params;
+  const { currentResource } = useContext(AppContext);
+  const [confirmInstall, setConfirmInstall] = useState<boolean>(false);
+
+  const userEntity = currentResource.entities.find(
+    (entity) => entity.name.toLowerCase() === "user"
```

Comment:
Not sure we need to change this now, but remember that a coming task is to allow renaming the user entity so this hard-coded "user" can be problematic later. We can change it when we handle that rename task.

Reply:
just for now, to find  the user entity, still need to decide what will be the logic/ configuration

Reply:
Okay, let's just move the string "user" to a const like in the server where you have USER_ENTITY_NAME